packages/sources/openexchangerates/src/endpoint/forex.ts

```diff
@@ -30,7 +52,9 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
   }
 
   const response = await Requester.request(options)
+  if (Array.isArray(to)) return handleBatchedRequest(jobRunID, request, response, 'rates', to)
```

Comment:
Both `base` and `quote` are batchable. This conditional should check for either being an array.

Reply:
Only quote / to is batchable in this case